### PR TITLE
Organized Manipulator Subsystem and Changed getDoubles to putDoubles

### PR DESCRIPTION
**Pull request by 1000DollarBill**
Alphabetized/Organized everything in the Manipulator Subsystem.
Changed getDoubles to putDoubles in the constructer preferences for every subsystem.

### CODE REVIEW

**Comment by msoucy** [comment]
Warning - using `put` will *override* any preferences that currently exist. I recommend making a function (you could probably stick it in `RobotMap.Preferences`) called `setDefault`, that checks `Preferences.containsKey()`, and then only sets the value if the key doesn't exist in the preferences.

File: src/main/java/frc/team166/robot/subsystems/Manipulator.java [hunk 1]
```diff
@@ -226,30 +251,40 @@ protected void interrupted() {
...
             }
 
-            @Override
-            protected boolean isFinished() {
+        };
+    }
 
-                return onTarget();
+    public Command ReEnablePotentiometer() {
+        return new SubsystemCommand("Re-Enable Potentiometer", this) {
```
Comment (bot190) [inline src/main/java/frc/team166/robot/subsystems/Manipulator.java]: This should be an action command that just calls enable

**Review by bot190 — COMMENTED** [review]
Looks good, just one comment

File: src/main/java/frc/team166/robot/subsystems/Lift.java [hunk 2]
```diff
@@ -98,14 +97,14 @@ public Lift() {
...
-        Preferences.getInstance().getDouble(RobotMap.Preferences.DOWN_MAX_SPEED, 1);
-        Preferences.getInstance().getBoolean(RobotMap.Preferences.USE_LIDAR, false);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_P, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_I, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_D, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_F, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.LIFT_UP_DOWN_INCREMENT, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.UP_MAX_SPEED, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.DOWN_MAX_SPEED, 1);
+        Preferences.getInstance().putBoolean(PreferenceStrings.USE_LIDAR, false);
```
Comment (bot190) [inline src/main/java/frc/team166/robot/subsystems/Lift.java]: Looks like you didn't get back to change this one.

**Review by bot190 — COMMENTED** [review]
Please change all of the Preferences.getInstance().putDouble to the new setDefaultDouble

File: src/main/java/frc/team166/robot/subsystems/Lift.java (hunk 2 above)
Comment (msoucy) [reply src/main/java/frc/team166/robot/subsystems/Lift.java]: that needs `setDefaultBool`

**Pull request merged**